README.md

```diff
@@ -15,19 +15,22 @@ pgai simplifies the process of building [search](https://en.wikipedia.org/wiki/S
 
 pgai brings embedding and generation AI models closer to the database. With pgai, you can now do the following directly from within PostgreSQL in a SQL query:
 
-* Create vector [embeddings](#embed) for your data.
-* Retrieve LLM [chat completions](#chat-complete) from models like Claude Sonnet 3.5, OpenAI GPT4o, Cohere Command, and Llama 3 (via Ollama).
-* Reason over your data and facilitate use cases like [classification, summarization, and data enrichment](docs/advanced.md) on your existing relational data in PostgreSQL.
+* [Create vector embeddings for your data](/docs/vectorizer-quick-start.md).
+* Retrieve LLM chat completions from models like [Claude Sonnet 3.5](/docs/anthropic.md), [OpenAI GPT4o](/docs/openai.md), [Cohere Command](/docs/cohere.md), and [Llama 3 (via Ollama)](/docs/ollama.md).
+* Reason over your data and facilitate use cases like [classification, summarization, and data enrichment](/docs/openai.md) on your existing relational data in PostgreSQL.
 
 Here's how to get started with pgai:
 
+* **TL;DR**:
+  * [Test drive automatic embedding vectorization](/docs/vectorizer-quick-start.md): quicky create embeddings using 
+     a pre-built Docker developer environment with a self-hosted Postgres instance with pgai, vectorizer, and vectorizer worker installed. 
```

Comment:
```suggestion
* **TL;DR**:
  * [Try out automatic embedding vectorization](/docs/vectorizer-quick-start.md): quicky create embeddings using 
     a pre-built Docker developer environment with a self-hosted Postgres instance with pgai and our vectorizer worker installed. (<10 minutes if you have docker installed!)
```

Not quite sure about the use of the term "test drive" here?
Test driven development does not seem relevant to the quick start, and is a loaded term so quite confusing imo.